docs/content/development/extensions-core/namespaced-lookup.md

```diff
@@ -13,14 +13,15 @@ Make sure to [include](../../operations/including-extensions.html) `druid-namesp
 ## Configuration
```

Comment:
adding `Cached namespace lookups all draw from the same cache pool, allowing each node to have a fixed cache pool that can be used by namespace lookups.`